pkg/cni/cni_test.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/containernetworking/cni/pkg/ns"
 )
 
+const (
+	dirMode = os.FileMode(0750)
```

Comment:
I think `os.ModeDir` is only used for checking perms - it's not actually required by `Mkdir*` (probably because the `os.ModeDir` is implicit in the call).